scripts/imgtool/main.py

```diff
@@ -282,8 +282,12 @@ def convert(self, value, param, ctx):
               help='Specify the value of security counter. Use the `auto` '
               'keyword to automatically generate it from the image version.')
 @click.option('-v', '--version', callback=validate_version,  required=True)
-@click.option('--align', type=click.Choice(['1', '2', '4', '8']),
+@click.option('--align', type=click.Choice(['1', '2', '4', '8', '16', '32',
+              '64', '128', '256', '512', '1024', '2048', '4096']),
               required=True)
+@click.option('--max-align', type=click.Choice(['1', '2', '4', '8', '16', '32',
```

Comment:
This extra option seems unnecessary. Makes `imgtool` less intuitive for the end user. We can calculate the value of `max_align`. It should be equal to `align` if the value of `align` is greater than 8 or otherwise it should be 8. Also, without the option it will be easier to refactor mcuboot code, if the need arises.